vendor/mruby/Makefile.am

```diff
@@ -12,7 +12,8 @@ DEFAULT_INCLUDES =						\
 	-I$(srcdir)/../mruby-source/src				\
 	-I$(srcdir)/../mruby-source/mrbgems/mruby-compiler/core	\
 	-I$(srcdir)/../mruby-source/mrbgems/mruby-io/include	\
-	-I$(srcdir)/../mruby-source/mrbgems/mruby-time/include
+	-I$(srcdir)/../mruby-source/mrbgems/mruby-time/include	\
+	-I$(builddir)/mruby-file-stat/src
```

Comment:
`$(builddir)/...` should be listed before `$(srcdir)/...`.

Reply:
I've fixed. Thanks!